src/overlays/actors/ovl_Boss_Va/z_boss_va.c

```diff
@@ -173,157 +319,3695 @@ static ColliderQuadInit D_8095BFF4 = {
+    } else if ((limbIndex >= 10) && (limbIndex < 20) && (sBodyBari[limbIndex - 10] != 0)) {
+        if (((limbIndex >= 16) || (limbIndex == 10)) && (sFightPhase <= PHASE_3)) {
+            gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_boss_va.c", 4208),
+                      G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            gSPDisplayList(POLY_XLU_DISP++, &gBarinadeDL_008BB8);
+        } else if ((limbIndex >= 11) && (sFightPhase <= PHASE_2)) {
+            gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_boss_va.c", 4212),
+                      G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            gSPDisplayList(POLY_XLU_DISP++, &gBarinadeDL_008BB8);
+        }
+
+        if (sCsState >= DEATH_START) {
+            sp78.x = Rand_CenteredFloat(530.0f);
+            sp78.y = Rand_CenteredFloat(530.0f);
+            sp78.z = -60.0f;
+        }
+        Matrix_MultVec3f(&sp78, &this->effectPos[limbIndex - 10]);
+    } else if (limbIndex == 25) {
+        gSPSegment(POLY_XLU_DISP++, 0x09,
+                   Gfx_TwoTexScroll(globalCtx->state.gfxCtx, 0, 0U, (globalCtx->gameplayFrames * 10) % 128, 0x10, 0x20,
+                                    1, 0, (globalCtx->gameplayFrames * 5) % 128, 0x10, 0x20));
+        gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_boss_va.c", 4232),
+                  G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        gSPDisplayList(POLY_XLU_DISP++, &gBarinadeDL_008D70);
+    } else if ((*dList != NULL) && (limbIndex >= 29) && (limbIndex < 56)) {
+        gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_boss_va.c", 4236),
+                  G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        gSPDisplayList(POLY_XLU_DISP++, *dList);
+    } else if ((limbIndex == 24) && (sCsState < DEATH_START)) {
+        sp78.x = (this->actor.shape.yOffset + 450.0f) + -140.0f;
+        Matrix_MultVec3f(&sp78, &this->unk_280);
+        sp78.x = 200.0f;
+        Matrix_MultVec3f(&sp78, &this->unk_274);
+    }
+
+    if ((limbIndex == 7) && (sCsState >= DEATH_START)) {
+        sp78.x = Rand_CenteredFloat(320.0f) + -250.0f;
+        sp78.y = Rand_CenteredFloat(320.0f);
+        sp78.z = Rand_CenteredFloat(320.0f);
+
+        if (sp78.y < 0.0f) {
+            sp78.y -= 150.0f;
+        } else {
+            sp78.y += 150.0f;
+        }
+
+        if (sp78.z < 0.0f) {
+            sp78.z -= 150.0f;
+        } else {
+            sp78.z += 150.0f;
+        }
+        Matrix_MultVec3f(&sp78, &this->unk_274);
+    }
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_boss_va.c", 4264);
+}
+
+s32 BossVa_SupportOverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                                   void* thisx) {
+    BossVa* this = THIS;
+
+    if (!this->onCeiling && (limbIndex == 4)) {
+        rot->z += this->headRot.x;
+    }
+    return 0;
```

Comment:
```suggestion
    return false;
```
`OverrideLimbDraw` returns a `doNotDrawLimb` boolean, same for other `OverrideLimbDraw` funcs in the file